# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/BrowserChild.cpp
# Commit: b326d33f98cb
# Full Hash: b326d33f98cbc1d08d8ca10609ae24e8c852767c
# Author: Andreas Farre <farre@mozilla.com>
# Date: 2020-08-01 09:48:10
# Regressor Bug: 1613431
# File Overlap Count: 1
# Description:
#   Bug 1613431 - Part 2: Ignore synced setters return value. r=nika
#   
#   Depends on D83645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D83646
# ==============================================================================

diff -r 4d29677fb419 -r b326d33f98cb dom/ipc/BrowserChild.cpp
--- a/dom/ipc/BrowserChild.cpp	Fri Jul 31 13:36:45 2020 +0000
+++ b/dom/ipc/BrowserChild.cpp	Fri Jul 31 13:37:00 2020 +0000
@@ -562,7 +562,8 @@
 
   // Set SANDBOXED_AUXILIARY_NAVIGATION flag if this is a receiver page.
   if (!PresentationURL().IsEmpty()) {
-    mBrowsingContext->SetSandboxFlags(SANDBOXED_AUXILIARY_NAVIGATION);
+    // Return value of setting synced field should be checked. See bug 1656492.
+    Unused << mBrowsingContext->SetSandboxFlags(SANDBOXED_AUXILIARY_NAVIGATION);
   }
 }
 